### ISSUE TITLE
About: a few things.

### ISSUE THREAD

**Issue by moshpirit** [redsolution/xabber-android#505]
Maybe it should be really good to change the "About" into a FAQ and maybe call it "Help" or "Help and About" or something like that and change a little the actual parts using it as a Welcome screen too:

### What's Xabber?

Xabber is an **open-source XMPP messenger for Android platform**. It is built around open standards, interoperability, design and user experience. Xabber supports many popular XMPP extension protocols, Off-The-Record Chat Encryption and is available in multiple languages.

### Wait, what's XMPP?

**Note**: I've used the [xmpp.org definition](https://xmpp.org/about-xmpp/), which I think it's more official.

The **Extensible Messaging and Presence Protocol** (XMPP) is an open technology for real-time communication, which powers a wide range of applications including instant messaging, presence, multi-party chat, voice and video calls, collaboration, lightweight middleware, content syndication, and generalized routing of XML data.

The protocol was originally named **Jabber**, and was developed by the Jabber open-source community in 1999 for near real-time instant messaging (IM),presence information, and contact list maintenance.

### Why should I use it?

XMPP is **highly extensible**, via extensions known as XEPs (XMPP Extension Protocol). Xabber supports a number of popular XEPs that are essential to providing great chat experience for our users.

Although the core technology is **stable**, the XMPP community continues to define various XMPP extensions through an open standards process run by the XMPP Standards Foundation. There is also an **active community of open-source and commercial developers**, who produce a wide variety of XMPP-based software.

### Why should I trust Xabber?

Xabber is **free software**, so the code it's totally open and everyone can see, modify and distribute it as they want. It's also economically free, so **you'll never have to pay for it** and it's advertisements-free. Xabber also allows us to **encrypt our messages** by using the Off-The-Record (OTR) Chat Encryption. But one of the best reasons is because it uses XMPP, which allows you to use the server that you most like so you have **total and real privacy**.

### How can I contribute if I'm a developer?

Xabber was originally developed by [Redsolution, Inc.](http://www.redsolution.com) — an international software and services company based in Russia and United States. Xabber was licensed under GNU/GPL v.3 license early in 2013. Since then, a number of individuals joined Xabber as developers, testers and translators.

Our goal is to create a stable, reliable and user friendly ecosystem for instant messaging that does not rely on proprietary services. We welcome anyone who believes in open standards and free information interchange to take part in moving Xabber forward.

Visit [our website](http://www.xabber.com/), send us an email to info@xabber.com and follow us on [Twitter](https://twitter.com/xabber_xmpp) and [Github](https://github.com/redsolution/xabber-android).

### What if I'm not a developer? 

Xabber is available in multiple languages thanks to many fine people from all over the world. We have created a [special page](https://github.com/redsolution/xabber-android/wiki/Top-translators) on our website to acknowledge their efforts.

You may join their ranks and help us improve translation quality by [contributing](https://crowdin.com/project/xabber/) your translations to Xabber project.

### License

Xabber source code is licensed under GNU GPL v.3 license and is available on GitHub.

Graphical images are licensed under Creative Commons Attribution Share-Alike (CC BY-SA) license.

Xabber & Redsolution logo are property of Redsolution, Inc.

More detailed information on Xabber [license file](https://github.com/redsolution/xabber-android/blob/master/LICENSE)

-------------------

**Note**: Maybe it could also include some "how can I donate some money to this beautiful project?" or something like that and all the pictures that it also uses (or even more to make it even more visual) and maybe some "more info" links.

**Comment by andrewnenakhov** [comment]
now you see why we didn't add 'about' screen to translations? )

**Comment by moshpirit** [comment]
Yes, I do, but then, we should discuss it and select a few months "about". After these few months (maybe 4 or 5) review it again and modify it if it's necessary. After that, the next review would be after a year and this "about" should be definitive
